broker/lib/jobs/BnRStatusPollerJob.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
```

Comment:
Just a question , why is this job called `BnRStatusPollerJob` ? Shouldnt it be just `BackupStatusPollerJob` ? For restore we depend upon CF , correct ?

Reply:
Yes, today we do. But as in SF 2.0 async call dependency on platform is removed, and if the same status poller is enhanced for restore, that's why kept the name like that.

Reply:
ok , then in that case I think we need to make few logs / variables etc. generic . , will comment.